srcpkgs/snapper/template

```diff
@@ -1,31 +1,30 @@
 # Template file for 'snapper'
 pkgname=snapper
-version=0.8.14
-revision=2
+version=0.8.15
+revision=1
 build_style=gnu-configure
-configure_args="--disable-zypp --with-conf=/etc/conf.d"
+configure_args="--disable-zypp --disable-systemd --with-conf=/etc/conf.d"
 conf_files="/etc/conf.d/snapper"
 make_dirs="/etc/snapper/configs 0755 root root"
 hostmakedepends="automake docbook-xsl libtool libxml2-devel libxslt
  gettext pkg-config"
 makedepends="acl-devel boost-devel dbus-devel e2fsprogs-devel libbtrfs-devel
- libmount-devel libxml2-devel pam-devel"
+ libmount-devel libxml2-devel pam-devel ncurses-devel ncurses-libtinfo-devel
+ json-c-devel"
 depends="dbus"
 short_desc="Tool for Linux filesystem snapshot management"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only"
 homepage="http://snapper.io"
 distfiles="https://github.com/openSUSE/snapper/archive/v${version}.tar.gz"
-checksum=d3abe4d542dade06b361e7c89b5de03bb5202853bc5e314ca74080caa24923f6
+checksum=fe0e32912275713f8fad17dbe510f32dbba8526365a849e3f7d6c786d43cfca2
 lib32disabled=yes
+LDFLAGS="-ljson-c"
 
 case "$XBPS_TARGET_MACHINE" in
-	*-musl)	# We define MUSL_LIBC to pick the correct return type
-		# for strerror_r(2) which is int (XSI-compliant)
-		CXXFLAGS="-DMUSL_LIBC=1"
-		# We also link against libintl
+	*-musl)
 		makedepends+=" gettext-devel"
-		LDFLAGS="-lintl"
+		LDFLAGS+=" -lintl"
 esac
```

Comment:
Could you try exporting `LIBS="-lintl"` only for musl? Having to put `-ljson-c` there explicitly seems wrong.

Reply:
And if possible switch to `"$XBPS_TARGET_LIBC" = musl`

Reply:
At first I thought we didnt need to link against json-c, since `pkg-config` is a dependency. But the build fails without it for some reason

Reply:
Found the issue. Snapper doesn't link to json-c if `--disable-zypp` is enabled. I will backport openSUSE/snapper#617 as well

Reply:
Wonderful